psi4/src/export_misc.cc

```diff
@@ -29,10 +29,13 @@
 #include "psi4/pybind11.h"
 
 #include "psi4/libciomr/libciomr.h"
+#include "psi4/libqt/qt.h"
 
 using namespace psi;
 
 void export_misc(py::module &m) {
+    m.def("timer_on", timer_on, "Start timer with argument as label");
+    m.def("timer_off", timer_off, "Stop timer of label argument");
```

Comment:
I think this was added in the SAPT(DFT) timer PR? Will cause a conflict.

Reply:
True. It's actually in the scfitertopy PR, three. But all need it. This one has docstrings!